app/build.gradle

```diff
@@ -98,21 +103,22 @@ android {
 
     leak {
       dimension "mode"
+      applicationIdSuffix ".leak"
       matchingFallbacks = ['internal', 'prod']
     }
 
     fairy {
       dimension "mode"
-      applicationIdSuffix "testfairy"
-      versionName "9.3.0." + getDate()
+      applicationIdSuffix ".testfairy"
+      versionName defaultConfig.getVersionName() + "" + getDate()
```

Comment:
is this needed? ``` + ""```

Reply:
defaultConfig.getVersionName() will always return string so it is not necessary - fixed.